webpack.config.js

```diff
@@ -42,11 +42,16 @@ module.exports = (env, args = {}) => {
   const showProfile = args.profile || false;
   const globalVariables = {
     'process.env': {
-      NODE_ENV: JSON.stringify(isProduction ? 'production' : 'development'), // Needed by vendor scripts
-      IS_STYLEGUIDE_BUILD: JSON.stringify(isStyleguideBuild),
-      HAS_WATCHER: hasWatcher,
-      BUILD_TIMESTAMP: new Date().getTime(),
+      'NODE_ENV': JSON.stringify(isProduction ? 'production' : 'development'), // Needed by vendor scripts
+      'IS_STYLEGUIDE_BUILD': JSON.stringify(isStyleguideBuild),
+      'HAS_WATCHER': hasWatcher,
+      'BUILD_TIMESTAMP': new Date().getTime(),
     },
+    '__VUE_OPTIONS_API__': true,
```

Comment:
Is this the new Vue3 config? Is there an URL you can `@see` to, for more details?